pkg/secrets/secrets.go

```diff
@@ -52,6 +52,22 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 		},
 		Data: map[string][]byte{
 			"userData": []byte(`<powershell>
+			function Get-RandomPassword {
+				Add-Type -AssemblyName 'System.Web'
+				return [System.Web.Security.Membership]::GeneratePassword(16, 2)
+			}
+
+			# Check if the capi user exists, this will be the case on Azure, and will be used instead of Administrator
+			if((Get-LocalUser |Where-Object {$_.Name -eq "capi"})  -eq $null) {
+				# The capi user doesn't exist, ensure the Administrator account is enabled
+				$UserAccount = Get-LocalUser -Name "Administrator"
```

Comment:
What happens if there is no Administrator account?

Reply:
It would fail and continue trying to execute the script if both the Administrator account and capi user are missing.
Changed it to only try to enable the Administrator account if it exists.

Reply:
We should output an error if there is no capi and Administrator user. This hopefully we will somehow end up in the cloud console log.

Reply:
This line does actually write out an error if the user doesn't exist.
Its the reason I used `Where-Object` in the capi query, that one will not write out an error if capi isn't found.

Reply:
I'll add a comment anyway